charts/aws-efs-csi-driver/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: aws-efs-csi-driver
-version: 2.1.4
-appVersion: 1.3.2
+version: 2.1.5
+appVersion: 1.3.3
```

Comment:
What about image tag for helm? Will that be another PR?

Reply:
should be fixed